featuretests/storage_test.go

```diff
@@ -125,13 +127,9 @@ func (s *cmdStorageSuite) TestStorageList(c *gc.C) {
 	createUnitWithStorage(c, &s.JujuConnSuite, testPool)
 
 	expected := `
-[Storage]        
-Unit             Id      Location  Status   Message  
-storage-block/0  data/0            pending           
-
-[Volumes]
-Machine  Unit             Storage  Id   Provider Id  Device  Size  State    Message
-0        storage-block/0  data/0   0/0                             pending  
+[Storage]
+Unit             Id      Provider id  Size  Status   Message
```

Comment:
I think it would be useful to have a Type in here? "filesystem" or "volume"
The charm metadata specifies the type; it's something *I'd* want to see as a user